zerver/lib/actions.py

```diff
@@ -606,6 +607,13 @@ def do_send_messages(messages):
             ums.extend(ums_to_create)
         UserMessage.objects.bulk_create(ums)
 
+
```

Comment:
Would remove this blank line (and the one after the new block) to match the existing style.